src/pages/jobs.md

```diff
@@ -185,3 +192,147 @@ If you're interested in joining us, please email eric@shorebird.dev. You can
 also (and are encouraged to) hop on our [Discord](https://discord.gg/shorebird)
 and chat with us there. We do pretty much everything in the public so you can
 very much see what we're like without even needing to apply.
+
+## Developer Relations / Marketing Lead
```

Comment:
For marketing/posting reasons, wondering if each of these should be a sub page so we can have actual unique URL's for them.